src/webgpu/web_platform/copyToTexture/canvas.spec.ts

```diff
@@ -87,12 +99,8 @@ class F extends CopyToTextureUtils {
     // Blue
     ctx.fillStyle = 'rgba(0, 0, 255,' + alphaValue + ')';
     ctx.fillRect(0, rectHeight, rectWidth, height - rectHeight);
-    // Black
-    if (isLosePrecisionDstFormat) {
-      ctx.fillStyle = 'rgba(255, 255, 255, 0.0)';
-    } else {
-      ctx.fillStyle = 'rgba(0, 0, 0,' + alphaValue + ')';
-    }
+    // White
+    ctx.fillStyle = 'rgba(255, 255, 255,' + alphaValue + ')';
```

Comment:
optional nit (just thought of this): I think this would be slightly more readable as `rgba(255, 255, 255, ${alphaValue})`